### ISSUE TITLE
Use Azure SignalR services?

### ISSUE THREAD

**Issue by StefanJansson** [soernt/signalr_client#15]
Is it possible to use Azure SignalR with this component or do you have plans to implement this?

**Comment by MSchmack** [comment]
Yes it's possible, there is no difference in the signalr protocol.
